scenes/ui/options_menu/input_remapping.gd

```diff
@@ -40,6 +40,8 @@ func _ready() -> void:
 	remap_aborted.connect( temporary_text_timed.bind(Messages.ABORTED, tempTextDuration) )
 	remap_successful.connect( temporary_text_timed.bind(Messages.SUCCESSFUL, tempTextDuration) )
 	
+	tree_exiting.connect( save_mappings )
```

Comment:
So I've found that this is only called when you press the "close" button.

I can confirm it works if you set a mapping and close it via this button.

But, a user can also close the menu by pressing the "esc" key. If you press the esc key, this signal is not triggered, which causes the mapping to not be stored

Reply:
Otherweird behavior spotted:
if you press esc to close the menu the tree_exiting is not triggered, but if you reopen the menu and press the option menu, it is triggered.

Reply:
Fixed, now it saves from the options_menu script instead of from the mapping menu.